File: app/src/main/java/com/marlon/portalusuario/activities/PrivacyActivity.java
package com.marlon.portalusuario.activities;

import android.os.Bundle;

import android.webkit.WebView;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.marlon.portalusuario.R;


public class PrivacyActivity extends AppCompatActivity {

    Toolbar mToolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_privacy);

        WebView myWebView = findViewById(R.id.webview_privacy);
        myWebView.loadUrl("https://sites.google.com/view/marlondejesusmilanesrivero/pol%C3%ADtica-de-privacidad");

    }
}

File: app/src/main/java/com/marlon/portalusuario/activities/PrivacyActivity.kt
package com.marlon.portalusuario.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.marlon.portalusuario.databinding.ActivityPrivacyBinding

class PrivacyActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityPrivacyBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.webviewPrivacy.loadUrl("https://portalusuarioapp.blogspot.com/p/portal-usuario-mas-que-una-simple.html")
    }
}
